app/src/components/Project/Content/AllCardsContent.tsx

```diff
@@ -64,17 +66,50 @@ export default function AllCardsContent({
         </div>
       </div>
       {/* Cards */}
-      <div className='h-full overflow-y-scroll hide-scrollbar'>
+      <div className=' h-full overflow-y-scroll hide-scrollbar'>
         {isEmpty(filteredCards) ? (
           <div className='text-center mt-10 w-full text-gray-300'>No cards found</div>
         ) : (
-          <div className='grid grid-cols-1 lg:grid-cols-2 xl:grid-cols-3'>
-            {filteredCards
-              .sort(({ createdAt: a }, { createdAt: b }) => b - a)
-              .map((card, index) => (
-                <CardPreview card={card} key={'card-' + index} onClick={() => onChooseCard(card)} />
-              ))}
-          </div>
+          <>
+            {/*start section topics by-roya add   */}
+            <div className='w-full h-14 px-6 py-3 bg-stone-50 justify-between items-center inline-flex border-b-2 border-[#A3A1A7]'>
+              <div className="text-zinc-700 text-sm font-normal font-['Monaspace Argon Var']">
+                What's new • 11{' '}
+              </div>
+              <div className='justify-start items-center gap-2 flex'>
+                <RiArrowRightSLine />
+                <div className="text-zinc-700 text-sm font-normal font-['Monaspace Argon Var']">
+                  3/11
+                </div>
+                <RiArrowLeftSLine />
+              </div>
+            </div>
+            {/* end section topics by-roya  */}
+            <div className='grid grid-cols-1 lg:grid-cols-2 xl:grid-cols-3'>
+              {filteredCards
+                .sort(({ createdAt: a }, { createdAt: b }) => b - a)
+                .map((card, index) => (
+                  <CardPreview
+                    card={card}
+                    key={'card-' + index}
+                    onClick={() => onChooseCard(card)}
+                  />
+                ))}
+            </div>
+            {/*start section user's last by-roya add   */}
+            <div className='w-full h-14 px-6 py-3 bg-stone-50 justify-between items-center inline-flex border-y-2 border-[#A3A1A7]'>
+              <div className="text-zinc-700 text-sm font-normal font-['Monaspace Argon Var']">
+                All topics
+              </div>
+              <div className='justify-start items-end flex '>
+                <div className=" h-5 text-slate-400 text-sm font-medium font-['Monaspace Argon Var']">
+                  Last modified
+                </div>
+                <RiArrowDown />
+              </div>
+            </div>
+            {/* end section user's last by-roya  */}
```

Comment:
We need to render here all topics 😁 (We have a header above, but we render nothing)

Reply:
I add a new item box for topics